python/tvm/relay/op/strategy/generic.py

```diff
@@ -799,6 +799,36 @@ def sparse_dense_padded_strategy(attrs, inputs, out_type, target):
     raise NotImplementedError("sparse_dense_padded is only implemented for cuda")
 
 
+# sparse_add
+def wrap_compute_sparse_add(topi_compute):
+    """wrap sparse add topi compute"""
+
+    def _compute_sparse_add(attrs, inputs, out_type):
+        return [topi_compute(inputs[0], inputs[1], inputs[2], inputs[3])]
+
+    return _compute_sparse_add
+
+
+@override_native_generic_func("sparse_add_strategy")
+def sparse_add_strategy(attrs, inputs, out_type, target):
+    """sparse add generic strategy"""
+    logger.warning("sparse add is not optimized for this platform.")
```

Comment:
so you are adding no optimized implementations?

Reply:
@tkonolige : Sorry for late response was stuck in another . I think this is general convention for generic strategy.

Reply:
are you going to add optimized implementations in a follow on pr?

Reply:
Yes, there will be follow-up PRs for sparse add, I believe it will be taken care in those.